### DISCUSSION TITLE
Embed new line \n

### DISCUSSION THREAD

**Discussion by WaydeTheKiwi** [zfbx/zdiscord#61] [Q&A] [answered]
When I included \n in an embed command, It does not create a new line. Is there a way to create a new line using the embed command?

**Comment by zfbx** [comment, marked as answer]
yes, used `<br>` discord seems to do weird escaping when sending a `\n` line break so I added a special case

**Comment by sanchez1122** [comment]
<a href="https://realmshosting.com/">fivem</a> is the best gaming hosting platform on the market. We provide a reliable and transparent platform for you to run your own dedicated server. Run any game, from any genre, and have a dedicated host with guaranteed uptime.
